Streetcode/Streetcode.WebApi/Utils/WebParsingUtils.cs

```diff
@@ -23,14 +23,9 @@ public WebParsingUtils(StreetcodeDbContext streetcodeContext)
 
         public async Task DownloadAndExtractAsync(string fileUrl, string zipPath, string extractTo, CancellationToken cancellationToken, ILogger logger = null)
```

Comment:
- Please separate method's arguments from each other, making the line shorter than the maximum of 120 symbols.
- logger here is nullable by default, however, it is not reflected on its type.

Reply:
done